<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ResultTurn</class>
 <widget class="QWidget" name="ResultTurn">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1400</width>
    <height>800</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLabel" name="label_background">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>130</y>
     <width>49</width>
     <height>14</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_resultIcon_0">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>730</y>
     <width>49</width>
     <height>14</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_resultIcon_1">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>730</y>
     <width>49</width>
     <height>14</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_resultIcon_2">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>730</y>
     <width>49</width>
     <height>14</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_resultIcon_3">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>730</y>
     <width>49</width>
     <height>14</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_resultIcon_4">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>730</y>
     <width>49</width>
     <height>14</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_resultIcon_5">
   <property name="geometry">
    <rect>
     <x>420</x>
     <y>730</y>
     <width>49</width>
     <height>14</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
